src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -203,6 +203,17 @@ function check_root_account_owns_user(root_account, account) {
 }
 
 
+/**
+ * @param {object} account
+ */
+function check_root_account(account) {
```

Comment:
you mean is_root_account

Reply:
I don't like functions that start with `_is` (this prefix I save for a boolean), and I prefer an action within the name.
So I can change it to `check_is_root_account`.

Reply:
perhaps you have something against it, but is_XXX is the common standard pattern for both boolean variables and boolean functions. check_XXX is a more commonly a pattern used when you need to do actual work to verify some expectation, which also typically can throw to report the check failed.